frontends/web/public/trash.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
For logos, we tend to use font-awesome. Let's do that instead here. Here's an example:
https://github.com/facebookresearch/dynabench/blob/dc1e435a2a01cb2cd15bbeccfad09897db2738b8/frontends/web/src/containers/TaskPage.js#L132
You should be able to find a trash can icon with font awesome:
https://fontawesome.com/v5.15/icons?d=gallery&p=2&q=delete&m=free